handlers.go

```diff
@@ -153,11 +140,6 @@ func createDbTableHandler(req *http.Request) (int, interface{}) {
 	return notImplemented()
 }
 
-// replaceDbTables handles PUT requests on /db/_schema .
-func replaceDbTablesHandler(req *http.Request) (int, interface{}) {
-	return notImplemented()
-}
-
 // updateDbTables handles PATCH requests on /db/_schema .
 func updateDbTablesHandler(req *http.Request) (int, interface{}) {
```

Comment:
It would better if you could avoid using interface{} for return types of those handlers. Or at least define a customized type as interface{} like `type response interface{}`

To improve the code, You can either:
1) return a customized response with an error
then you funcs are like `func someHandler(req *http.Request) response, error`
A good response can have a code to mark success, with a non-nil data.

```
type response struct {
    Code int
    Data interface{}
}
```
2) just return a MyResponse struct:
```
type MyResponse struct {
    Code int
    Err error
    Data interface{}
}
```
then you funcs are like `func someHandler(req *http.Request) MyResponse`

In either way, you should follow this order `check err -> check code -> check data` in callers.
Personally I prefer the 1st way, it's more go-style.